### ISSUE TITLE
can we have set it as network adaptor?

### ISSUE THREAD

**Issue by jacob755** [extremecoders-re/go-dispatch-proxy#30]
hello
can we set it in virtual network adaptor interface so get one unique local ip??  ?
now with @ we define how many connection use for each interface and when we visit site its see our ip change i dont want it happen can we do something?

**Comment by extremecoders-re** [comment]
Yes you can create virtual network adapters (like tun/tap on linux) and bind to that interface instead.

**Comment by jacob755** [comment]
maybe like this!? i searched github and found these i am unable to set it its not easy.help appreciated 
or even maybe add  them to app thanks
https://github.com/xjasonlyu/tun2socks
https://github.com/FlowerWrong/tun2socks
https://github.com/eycorsican/go-tun2socks
https://github.com/dosgo/goSocksTap
